hack/verify-typecheck.sh

```diff
@@ -30,8 +30,12 @@ cd "${KUBE_ROOT}"
 
 make --no-print-directory -C "${KUBE_ROOT}" generated_files
 
+# As of June, 2020 the typecheck tool is written in terms of go/packages, but
+# that library doesn't work well with multiple modules.  Until that is done,
```

Comment:
it's not clear in this sentence until what is done.

Reply:
Yeah, I'm a little confused why `run-in-gopath` in still needed.

Reply:
Our tree has multiple `go.mod` in it.  We're making a tool that does filesystem walk, which ignores go.mod.  These are not compatible,

We have to choose - we can ignore go.mod, in which case it has to be under a GOPATH or we can respect go.mod which requires that we run this tool multiple times against each module (or something, TBD).

I chose to rewrite to go/packages first.  Hence GOPATH

Reply:
I'd suggest to expand the comment a little bit. When reading it out of context it's unclear what this `Until that` refers to =). Maybe "Until it works well with multiple modules,"